BibBuddy/app/src/main/res/layout/fragment_notes.xml

```diff
@@ -7,26 +7,18 @@
     android:layout_marginBottom="150dp"
     android:translationY="80dp">
 
-    <RelativeLayout
-        android:id="@+id/hidden_delete_panel"
+    <TextView
+        android:id="@+id/empty_notes_list_view"
         android:layout_width="match_parent"
-        android:layout_height="70dp"
-        android:background="@color/alert_red"
-        android:translationY="495dp"
-        android:translationZ="12dp"
-        android:visibility="gone">
-
-        <ImageButton
-            android:id="@+id/panel_delete"
-            android:layout_width="50dp"
-            android:layout_height="50dp"
-            android:layout_centerInParent="true"
-            android:background="@null"
-            android:scaleType="fitCenter"
-            android:src="@drawable/delete"
-            android:tint="@color/true_white" />
-
-    </RelativeLayout>
+        android:layout_height="match_parent"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="58dp"
+        android:layout_marginLeft="20dp"
+        android:layout_marginRight="20dp"
+        android:text="@string/empty_notes_list"
```

Comment:
I think for the sake of consistency throughout the app we should use the neutral string "empty_list" here - that notes are meant here is clear from the fragment itself.

```suggestion
        android:text="@string/empty_list"
```

Reply:
Done.